bin/git-bulk

```diff
@@ -36,8 +36,12 @@ function addworkspace {
             source=$(realpath "$source" 2>/dev/null)
             if [ -f "$source" ]; then
                 pushd "$wsdir" > /dev/null
-                while read -r line; do git clone "$line"; done < "$source";
-                popd > /dev/null
+                    while IFS= read -r line || [[ -n $line ]]; do
+                      if [ -n "$line" ]; then
```

Comment:
Would you honor the original indentation? Thanks!

Reply:
Done